frontend/workflows/projectSelector/src/hello-world.tsx

```diff
@@ -206,6 +207,52 @@ const deriveSwitchStatus = (state: State, group: Group): boolean => {
   );
 };
 
+const StyledCount = styled.span({
+  color: "rgba(13, 16, 48, 0.6)",
+  backgroundColor: "rgba(13, 16, 48, 0.03)",
+  fontVariantNumeric: "tabular-nums",
+  letterSpacing: "2px",
+  borderRadius: "4px",
+  fontWeight: "bold",
+  fontSize: "12px",
+  padding: "1px 2px 1px 4px",
+  margin: "0 4px",
+});
+
+const StyledTitle = styled.span({
+  fontWeight: 500,
+});
+
+const MenuItem = styled.div({
+  display: "flex",
+  alignItems: "center",
+  "&:hover": {
+    backgroundColor: "rgba(13, 16, 48, 0.03)",
+  },
+  "&:hover > span": {
+    display: "inline-flex", // Unhide "only" hidden span.
+  },
+});
+
+const StyledHeader = styled.div({
```

Comment:
nit
```suggestion
const StyledProjectHeader = styled.div({
```

Reply:
these nits arent important but I caught myself using the name names (title, header) in my PR. but once we break up the file into components, it will be cleaner/easier to read overall